src/components/AboutUs.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./About_us.css";
+
+export default function AboutUs() {
+  return (
+    <footer>
```

Comment:
Footer is a part of your main component, you shouldn't use it in all other components

Reply:
@discrimine asd